src/main/java/com/minecolonies/rankup/modules/databases/command/convertToConfCommand.java

```diff
@@ -0,0 +1,96 @@
+package com.minecolonies.rankup.modules.databases.command;
+
+import com.minecolonies.rankup.internal.command.RankupSubcommand;
+import com.minecolonies.rankup.modules.core.config.AccountConfigData;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.scheduler.Task;
+import org.spongepowered.api.text.Text;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class convertToConfCommand extends RankupSubcommand
+{
+    @Override
+    protected String[] getAliases()
+    {
+        return new String[] {"database-to-conf"};
+    }
+
+    @Override
+    protected Optional<Text> getDescription()
+    {
+        return Optional.of(Text.of("None"));
+    }
+
+    @Override
+    protected Optional<String> getPermission()
+    {
+        return Optional.empty();
+    }
+
+    @Override
+    public CommandResult execute(final CommandSource src, final CommandContext args) throws CommandException
+    {
+        Task.Builder taskBuilder = Sponge.getScheduler().createTaskBuilder();
+        taskBuilder.name("conf-to-database").async().delayTicks(4L).execute(this::convert).submit(getPlugin());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 4L to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS109)